### ISSUE TITLE
Localization of unspecified epileptiform EEG abnormalities

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6721]
**Preferred term label:**
Please consider creating the following 6 terms because often the localization of epileptiform features on EEG are reported without knowledge of the specific form of epileptiform feature. Hence these would rescue some localization value from raw data missing the specific type of epileptiform abnormality.

EEG with frontal epileptiform discharges
EEG with temporal epileptiform discharges
EEG with central epileptiform discharges
EEG with parietal epileptiform discharges
EEG with occipital epileptiform discharges
EEG with centrotemporal epileptiform discharges

Each should be the dual parent of the child of each of the following that corresponds to the same localization:

EEG with focal spikes HP:0011193
EEG with focal spike waves HP:0011197
EEG with focal sharp waves HP:0011196
EEG with focal sharp slow waves HP:0011195

Not all of these have EEG with centrotemporal localization.

**Synonyms**
None

**Definition (free text, please give PubMed ID)**
For each, of the form:

"Focal epileptiform EEG discharges recorded in the [insert localization] region."

**Parent term (use hpo.jax.org/app)**
All would be children of _EEG with focal epileptiform discharges HP:0011185_

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Most epilepsies with focal features

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
Thanks, added these terms!
